server/routes/checkAnswers.js

```diff
@@ -128,37 +131,38 @@ const createDetailsObj = (details = {}) => {
   return {
     'Positive communication used': details.positiveCommunication,
     'Personal Protection Techniques': details.personalProtectionTechniques,
-    'Baton Drawn': details.batonDrawn + (details.batonUsed === 'Yes' ? ' - and used' : ' - but not used'),
-    'Pava drawn': details.pavaDrawn + (details.pavaUsed === 'Yes' ? ' - and used' : ' - but not used'),
-    'Guiding hold used':
-      details.guidingHold +
-      (details.guidingHold === 'Yes' ? howManyOfficersInvolved(details.guidingHoldOfficersInvolved) : ''),
-    'Restraint used':
-      details.restraint +
-      (details.restraint === 'Yes' ? ` - ${getRestraintPositions(details.restraintPositions)}` : ''),
-    'Handcuffs used':
-      details.handcuffsApplied + (details.handcuffsApplied === 'Yes' ? typeOfHandcuffsUsed(details.handcuffsType) : ''),
+    'Baton drawn': whenPresent(details.batonDrawn, value =>
+      value === 'Yes' ? wasWeaponUsed(details.batonUsed) : 'No'
+    ),
+    'PAVA drawn': whenPresent(details.pavaDrawn, value => (value === 'Yes' ? wasWeaponUsed(details.pavaUsed) : 'No')),
+    'Guiding hold used': whenPresent(details.guidingHold, value =>
+      value === 'Yes' ? `Yes ${howManyOfficersInvolved(details.guidingHoldOfficersInvolved)}` : 'No'
```

Comment:
meh.. these methods are a bit inconsistent as to whether they will have `Yes` or `Yes - ` or `` as part of the prefix. 
Kind of prefer the more explicit approach of moving `Yes -` up to this level, like the `getRestraintPositions` example but not precious.